src/crab/ebpf_transformer.cpp

```
@@ -1049,13 +1049,15 @@ void EbpfTransformer::operator()(const LoadMapAddress& ins) {
 void EbpfTransformer::assign_valid_ptr(const Reg& dst_reg, const bool maybe_null) {
     using namespace dsl_syntax;
     const RegPack& reg = reg_pack(dst_reg);
+    dom.state.values.havoc(reg.svalue);
     dom.state.values.havoc(reg.uvalue);
     if (maybe_null) {
```

Clang (14.0.6)

[warning] if with identical then and else branches

(bugprone-branch-clone)